sklearn/compose/tests/test_column_transformer.py

```diff
@@ -2354,6 +2355,35 @@ def test_column_transformer__getitem__():
         ct["does_not_exist"]
 
 
+@pytest.mark.parametrize("transform_output", ["default", "pandas"])
```

Comment:
and polars?

Reply:
nop because polars does not allow to create a dataframe with integer columns. It will require string and thus we cannot trigger the regression.